common/djangoapps/static_replace/__init__.py

```diff
@@ -78,7 +78,7 @@ def replace_course_urls(text, course_key):
     Replace /course/$stuff urls with /courses/$course_id/$stuff urls
 
     text: The text to replace
-    course_module: A CourseBlock
+    course_block: A CourseBlock
```

Comment:
```suggestion
    course_key: a CourseKey
```

Nit: while your replacement is correct in the context of the PR, we could use this opportunity to fix the docstring.

Reply:
Oh, I didn't notice this, thanks.